tests/plugins/elementspath/manual/elementspathdraggable.html

```diff
@@ -0,0 +1,16 @@
+<textarea id="editor1">
```

Comment:
I think the file name could be simplified. Look, it's already in `elementspath` plugin tests so it doesn't necessarily need to have this part in its name.

How about simply `dragging`?